indico/modules/events/contributions/client/js/index.jsx

```diff
@@ -17,7 +17,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import 'indico/modules/events/util/types_dialog';
-import EditableSubmissionButton from 'indico/modules/events/editing/components/EditableSubmissionButton';
+import EditableSubmissionButton from 'indico/modules/events/editing/editing/EditableSubmissionButton';
```

Comment:
I know it is because we strip `client/js` but this looks so wrong :P

Reply:
yeah, i thought the same thing when writing this... ;x